components/buttons.css

```diff
@@ -0,0 +1,79 @@
+.buttons {
```

Comment:
This name is confusing as this is a section and not a button. How about:
```suggestion
.energy-rating {
```
or something like that?